rescheduler/rescheduler.go

```diff
@@ -67,6 +67,10 @@ var (
 	podScheduledTimeout = flags.Duration("pod-scheduled-timeout", 10*time.Minute,
 		`How long should rescheduler wait for critical pod to be scheduled
 		 after evicting pods to make a spot for it.`)
+
+	podBucketList = flags.StringSlice("bucket-list", []string{},
```

Comment:
I like that you're not creating the "other" bucket. What about more descriptive names, though?
The flag could be called "valuable-namespaces". Eventually we could have another "avoid-namespaces" or "namespace-blacklist" to prevent specific namespaces such as prod from ever getting touched. The variable could have a similar name, valuableNSList. And the description of the flag should mention what happens to pods in namespaces not in this list (they get first on the candidate list..).